src/Components/Modal/Modal.tsx

```diff
@@ -44,17 +49,30 @@ export class Modal extends React.Component<ModalProps, any> {
   }
 
   render(): JSX.Element {
-    const newProps: any = { ...this.props }
-    delete newProps.onClose
-    delete newProps.show
+    const { children, show } = this.props
+
+    const transitions = show
+      ? {
+          from: { opacity: 0, transform: "translate(-50%,-40%)" },
+          to: { opacity: 1, transform: "translate(-50%,-50%)" },
+        }
+      : { from: { opacity: 1 }, to: { opacity: 0 } }
 
-    if (!this.props.show) {
-      return null
-    }
     return (
       <div>
-        <ModalContainer {...newProps}>{this.props.children}</ModalContainer>
-        <Overlay onClick={this.close} />
+        <Spring {...transitions as any}>
+          {(styles: any) => (
```

Comment:
yes, I've been investigating the library for a bit. cross-platform support was one main reason why I went that route especially since we want modals on modal as well.